### DISCUSSION TITLE
"Flat" m2o interface

### DISCUSSION THREAD

**Discussion by d1rOn** [directus/directus#17023] [Feature Requests]
The idea of this interface is to display all fields from One collection in list of Many collection fields, with no drawer using.
Thomething similar like Translations interface working.
Additional options of this interface might be overriding of visibility of One collection fields.

Advantages:
1. Reduce nested drawers count.
2. We can create subcollections with basic functionalities and then reuse them whenever we need, and they will look like native fields.

**Comment by ghost** [comment]
@d1rOn I would love to see this. I think it’s overdue, because there is no method to [reuse fields](https://github.com/directus/directus/discussions/17160) at the moment.

A classic use case would be meta info fields (for seo and link preview) like `meta_title`, `meta_description` and `meta_image`, which I want to reuse on multiple collections.

> Additional options of this interface might be overriding of visibility of One collection fields.

I would definitely override values for `required`, `unique`, validation, conditions and everything else that would make sense for a flat interface.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
